test/dot-index-plugin/dot-index-plugin.test.js

```diff
@@ -6,8 +6,8 @@ const fs = require('fs-extra')
 const to = relPath => path.resolve(__dirname, relPath)
 
 test('Generates dot index files', end => {
-  
   const config = {
+    mode: 'production',
```

Comment:
Webpack recommends always adding `mode`

https://webpack.js.org/migrate/5/#make-sure-to-use-mode